tenacity/tests/test_tenacity.py

```diff
@@ -382,7 +382,7 @@ def go(self):
 
 
 class NoIOErrorAfterCount(object):
-    "Holds counter state for invoking a method several times in a row."
+    """Holds counter state for invoking a method several times in a row."""
```

Comment:
I normally would separate the unique changes like I did in #80, but I don't like making style only pull requests as it simply makes git blame harder to read.

Those style changes make sense in that commit IMO. Since the docstrings were copied in the complement functions, we made their style match. And I didn't alter anything I didn't directly copy/change. For example there is still non-triple quoted docstrings for `NoneReturnUntilAfterCount`.

If you prefer style only pull requests I can split it off. If I wasn't adding `IOErrorUntilCount` with a matching docstring to this function I would've never changed this.